src/main/java/com/atlassian/bitbucket/jenkins/internal/client/BitbucketFilePathClientImpl.java

```diff
@@ -30,6 +31,24 @@ public BitbucketFilePathClientImpl(BitbucketRequestExecutor bitbucketRequestExec
         this.repositorySlug = repositorySlug;
     }
 
+    @Override
+    public Type getFileType(String path, String ref) {
+        HttpUrl url = getUrlBuilder(path, ref).addQueryParameter("type", "true").build();
```

Comment:
The `type` query parameter specifies that we don't need the file content and simply return and object with the file type. example response.
```json
{
   "type": "file"
}
```